lib/webhooks/register.ts

```diff
@@ -167,21 +166,25 @@ function buildHandlerFromNode(edge: WebhookCheckResponseNode): WebhookHandler {
       break;
     case 'WebhookEventBridgeEndpoint':
       handler = {
-        id: edge.node.id,
         deliveryMethod: DeliveryMethod.EventBridge,
         arn: endpoint.arn,
       };
       break;
     case 'WebhookPubSubEndpoint':
       handler = {
-        id: edge.node.id,
         deliveryMethod: DeliveryMethod.PubSub,
         pubSubProject: endpoint.pubSubProject,
         pubSubTopic: endpoint.pubSubTopic,
       };
       break;
   }
 
+  // Set common fields
+  handler.id = edge.node.id;
+  handler.includeFields = edge.node.includeFields;
+  handler.metafieldNamespaces = edge.node.metafieldNamespaces;
+  handler.privateMetafieldNamespaces = edge.node.privateMetafieldNamespaces;
```

Comment:
Wondering if these could be sorted when stored, so that at least that doesn't need to be done in the `categorizeHandlers` later